### DISCUSSION TITLE
建议：增加eqref{}关键字

### DISCUSSION THREAD

**Discussion by Nineteen-Hundred** [note286/xduts#31] [Q&A] [answered]
尝试了一下，发现好像目前还没有eqref{xxxx}，与ref{xxx}的差别就在于自动加上了外面的括号，例如效果直接为(2-1)，ref的话是2-1，如果作者有时间能加上的话就太好了，感谢！

**Comment by note286** [comment, marked as answer]
`\eqref`是`amsmath`宏包提供的功能。

```latex
\documentclass{xdupgthesis}
\usepackage{amsmath}
\begin{document}
\chapter{title}
\begin{equation}
\label{xxxx}
a+b=c
\end{equation}
\eqref{xxxx}
\end{document}
```

![image](https://user-images.githubusercontent.com/16207292/211007759-b34e70f2-26fa-4c77-97a0-253443b51a15.png)
